solution-base/mongodb/Makefile

```diff
@@ -3,10 +3,10 @@ CHART_DIR:="${ROOT_DIR}/charts"
 
 CHART_REPO:="https://charts.bitnami.com/bitnami"
 CHART_MONGO_REPLICASET_VERSION:="7.8.0"
```

Comment:
`CHART_MONGO_REPLICASET_VERSION:="7.8.0"` not used anymore, should be removed...
(left-over from non-sharded)